src/effect_reshade.cpp

```diff
@@ -332,14 +332,78 @@ namespace vkBasalt
             
             //pipeline
             
+	    // Configure effect
+            std::vector<VkSpecializationMapEntry> specMapEntrys;
+            std::vector<char> specData;
+
+            uint32_t spec_id = 0;
+            uint32_t offset = 0;
+
+            uint32_t bvalue, uvalue;
+            int32_t ivalue;
+            float fvalue;
```

Comment:
Why aren't these defined in the loop?

Reply:
Placed where `std::vector<std::variant<float,uint32_t,int32_t>>` was in my earlier attempt.
Better refactor this code late, if you going to accept it. Maybe into separate helper function. Currently it's just a hack.